examples/server/public/index.html

```diff
@@ -253,7 +257,7 @@ <h3 class="text-lg font-bold mb-6">Settings</h3>
     <label class="input input-bordered join-item grow flex items-center gap-2 mb-2">
       <!-- Show help message on hovering on the input label -->
       <div class="dropdown dropdown-hover">
-        <div tabindex="0" role="button" class="font-bold">{{ configKey }}</div>
+        <div tabindex="0" role="button" class="font-bold">{{ name }}</div>
```

Comment:
I think it would be better to call this `label` instead of `name`, because `name` is a reserved keyword for `<input>` element inside `<form>`

We can also make it optional. If it's unspecified, we use `configKey` as label:

```suggestion
        <div tabindex="0" role="button" class="font-bold">{{ label || configKey }}</div>
```